exercises/list-ops/canonical-data.json

```diff
@@ -0,0 +1,147 @@
+{
+  "#": [
+    "Though there are no specifications here for dealing with large lists,",
+    "implementers may add tests for handling large lists to ensure that the",
+    "solutions have thought about performance concerns."
+  ],
+  "append": {
+    "description": "append entries to a list and return the new list",
+    "cases": [
+      {
+        "description": "empty lists",
+        "list1": [],
+        "list2": [],
+        "expected": []
+      },
+      {
+        "description": "empty list to list",
+        "list1": [],
+        "list2": [1,2,3,4],
+        "expected": [1,2,3,4]
+      },
+      {
+        "description": "non-empty lists",
+        "list1": [1,2],
+        "list2": [2,3,4,5],
+        "expected": [1,2,2,3,4,5]
+      }
+    ]
+  },
+  "concat": {
+    "description": "concat lists and lists of list into a new list",
+    "cases": [
+      {
+        "description": "empty list",
+        "lists": [],
+        "expected": []
+      },
+      {
+        "description": "list of lists",
+        "lists": [[1,2],[3],[],[4,[5,6]]],
```

Comment:
> `expect( concat([ [1,2,3], [], [4, 5] ]) ) to equal [[1,2,3,4,5]]`

I would expect the answer to be `[1, 2, 3, 4, 5]` rather than `[[1, 2, 3, 4 ,5]]` - the function took a list of list of integers and turned it into a list of integers.

Or if we want to use an existing track's test as a model (don't know whether any other tracks have this exact one) https://github.com/exercism/xhaskell/blob/master/exercises/list-ops/test/Tests.hs#L112 - `[ [1, 2], [3], [], [4, 5, 6]]` and `[1, 2, 3, 4, 5, 6]`